draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -917,6 +917,13 @@ Example use cases include:
    all packets can be scheduled  over all paths leading to high capacity and
    and high energy efficiency.
 
+The default behavior is, that a path can always be used for packet scheduling 
+decisions (MP_PRIO=3), if the path has been established and added to an 
+existing MP-DCCP. Setting all paths to not use (MP_PRIO=0) MUST be avoided. 
```

Comment:
Perhaps it is better to say that "at least one path MUST have a MP_PRIO value greater than or equal to one" "at all times" or if we want it to be possible to suspend a connection "for it to be allowed to send on the connection".

Reply:
good idea - I agree with you proposal and tend to go for the 2nd option to enable temporary suspension of a path

Reply:
Good idea, i will update wrt Annas comments